src/schema/v2/sale_artwork.ts

```diff
@@ -145,6 +146,32 @@ export const SaleArtworkType = new GraphQLObjectType<any, ResolverContext>({
         description: "Singular estimate field, if specified",
         resolve: ({ estimate_cents }) => estimate_cents,
       },
+      formattedStartDateTime: {
+        type: GraphQLString,
+        description:
+          "A formatted description of when the lot starts or ends or if it has ended",
+        resolve: (saleArtwork, _options, { defaultTimezone, saleLoader }) => {
+          if (!!saleArtwork.sale) {
+            return formattedStartDateTime(
+              saleArtwork.sale.start_at,
+              saleArtwork.endedAt ||
+                saleArtwork.endAt ||
+                saleArtwork.sale.end_at,
+              null,
+              defaultTimezone || DEFAULT_TZ
```

Comment:
This reads odd, default or... default? What's the difference? (possibly `defaultTimezone` takes into account a default already). Or, if it represents the request's time zone, it should be called `requestTimezone`?

Reply:
[defaultTimeZone](https://github.com/artsy/metaphysics/blob/b6167e0d1142853203523198d889506563fb7195/src/index.ts#L153-L156) comes from the context and is set from a header from the response. It's called `defaultTimeZone` in the context globally but agree that's confusing. Should I change that? in `src/index.ts`?

Reply:
Yea, i think that name is confusing b/c there's no actual default taken into account. I would maybe rename it to be more explicit (`requestTimezone`), but also feel free to do that in a separate PR, or make a JIRA ticket (or GitHub issue!) to do it later. Not a blocker.

Reply:
ticket: https://artsyproduct.atlassian.net/browse/BID-252